### ISSUE TITLE
2.7.1 nfs-ganesha-ceph Debian package missing libfsalceph.so

### ISSUE THREAD

**Issue by jrosengren** [nfs-ganesha/nfs-ganesha#378]
```
# lsb_release -a
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 9.6 (stretch)
Release:        9.6
Codename:       stretch
```
```
# dpkg -l | grep ganesha
ii  nfs-ganesha                          2.7.1-1                        amd64        nfs-ganesha is a NFS server in User Space
ii  nfs-ganesha-ceph:amd64               2.7.1-1                        amd64        nfs-ganesha fsal ceph libraries
ii  nfs-ganesha-gluster:amd64            2.7.1-1                        amd64        nfs-ganesha fsal ceph libraries
```
```
# dpkg-query -L nfs-ganesha-ceph
/.
/usr
/usr/share
/usr/share/doc
/usr/share/doc/nfs-ganesha-ceph
/usr/share/doc/nfs-ganesha-ceph/changelog.Debian.gz
/usr/share/doc/nfs-ganesha-ceph/copyright
/etc/ganesha/ceph.conf
```

If I missed some documentation somewhere about this library being provided elsewhere or in a different package, my apologies.  Otherwise, is this just a package build oversight?

**Comment by christmart** [comment]
You're right. In the version on download.nfs-ganesha.org it is missing. The Debian version for buster on ftp.debian.org has the libraries included.

**Comment by kalebskeithley** [comment]
There's a separate issue tracker for Debian packaging issues: https://github.com/nfs-ganesha/nfs-ganesha-debian/issues

fixed in nfs-ganesha-2.7.1-2 packages on download.nfs-ganesha.org

**Comment by jrosengren** [comment]
I didn't know that other issue tracker existed - will use that next time.

Thank you!
